mongoid-locker.gemspec

```diff
@@ -18,5 +18,5 @@ Gem::Specification.new do |s|
   s.files = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(spec)/}) }
   s.require_paths = ['lib']
 
-  s.add_dependency 'mongoid', '>= 5.0', '< 8'
+  s.add_dependency 'mongoid', '>= 5.0', '< 8.1'
```

Comment:
Should we either remove this restriction and make it either < 9. There's no reason to prevent future 8.2 that would theoretically work according to semver.

Reply:
Sure. let make it < 9